code/items/equipments/C4Equipment.cs

```diff
@@ -0,0 +1,99 @@
+using Sandbox;
+
+using TTTReborn.Player;
+
+namespace TTTReborn.Items
+{
+    /// <summary>
+    /// C4 equipment definition, for the physical entity, see items/equipments/entities/C4Entity.cs
+    /// </summary>
+    [Library("ttt_c4"), Hammer.Skip]
+    public partial class C4Equipment : TTTEquipment, IBuyableItem
+    {
+        public override string ViewModelPath => "";
+        public override SlotType SlotType => SlotType.OffensiveEquipment;
+
+        private const int DropPositionOffset = 50;
+        private const int DropVelocity = 500;
+        private const int PlaceDistance = 200;
+
+        public int Price => 0;
+
+        public C4Equipment() : base()
+        {
+
+        }
+
+        public override void Spawn()
+        {
+            base.Spawn();
+
+            RenderAlpha = 0f;
+        }
+
+        public override void Simulate(Client client)
+        {
+            if (!IsServer)
+            {
+                return;
+            }
+
+            if (Owner is not TTTPlayer owner)
+            {
+                return;
+            }
+
+            using (Prediction.Off())
+            {
+                if (Input.Pressed(InputButton.Attack1)) //Drops C4
+                {
+                    _ = new C4Entity()
+                    {
+                        Position = Owner.EyePos + Owner.EyeRot.Forward * DropPositionOffset,
+                        Rotation = Owner.EyeRot,
+                        Velocity = Owner.EyeRot.Forward * DropVelocity,
+                    };
+                    (owner.Inventory as Inventory).Remove(this);
```

Comment:
Could just extract this logic into a method
`dropC4`, then you wouldn't need the comment.